scout/server/blueprints/cases/templates/cases/utils.html

```diff
@@ -531,18 +535,18 @@ <h6>No causative variants</h6>
 {% endmacro %}
 
 {% macro suspects_list(suspects, institute, case, manual_rank_options, cancer_tier_options) %}
+   {{ clinvar_howto_modal() }}
   <div class="card mt-3">
-    <div class="d-flex align-items-center" data-bs-toggle='tooltip' title="Displays all variants that has been pinned for this case">
-      <h6><span class="fa fa-map-pin ms-3 mt-2"></span>&nbsp;Pinned variants ({{suspects|length}})
-      </h6>
+    <div class="d-flex align-items-center" data-bs-toggle='tooltip' title="Displays all variants that has been pinned or submitted to ClinVar for this case">
+      <h6><span class="fa fa-map-pin ms-3 mt-2"></span>&nbsp;Variants pinned ({{suspects|length}}) or included in ClinVar ({{ case.clinvar_variants.items()|length }}) <a data-bs-target="#howto" href="#" data-bs-toggle="modal">?</a></h6>
```

Comment:
```suggestion
      <h6><span class="fa fa-map-pin ms-3 mt-2"></span>&nbsp;Variants pinned ({{suspects|length}}) or in a ClinVar submission ({{ case.clinvar_variants.items()|length }}) <a data-bs-target="#howto" href="#" data-bs-toggle="modal">?</a></h6>
```

You don't know if the submission object in Scout was actually submitted to ClinVar